.github/workflows/verify.yml

```diff
@@ -0,0 +1,55 @@
+name: Verify
+
+on:
+  push:
+    branches:
+      - '*'
+  pull_request:
+    branches:
+      - '*'
+
+jobs:
+  test:
+    runs-on: ubuntu-18.04
+    timeout-minutes: 40
+
+    strategy:
+      fail-fast: true
+      matrix:
+        ruby:
+          - 2.7
```

Comment:
Should we not also be running the tests with Ruby 2.5 and Ruby 2.6 since we need to use this library over in Framework?

Reply:
Sure; Done 🎉 

![image](https://user-images.githubusercontent.com/60357436/125097319-eb910780-e0cd-11eb-8f1a-c4bce1bf1e42.png)